lake/ztests/query-sql-agg.yaml

```
@@ -0,0 +1,22 @@
+# brimdata/brim#1884
+
+script: |
+  export ZED_LAKE_ROOT=test
+  zed lake init -q
+  zed lake create -q test
+  zed lake use -q test@main
+  zed lake load -q data.zson
+  zed lake query -z 'from test | SELECT count(), id GROUP BY id | sort id'
```

No need for `from` after `zed lake use`.
```suggestion
  zed lake query -z 'SELECT count(), id GROUP BY id | sort id'
```